usecases/objects/references_update.go

```diff
@@ -69,8 +69,8 @@ func (m *Manager) UpdateObjectReferences(ctx context.Context, principal *models.
 	}
 	defer unlock()
 
-	validator := validation.New(m.vectorRepo.Exists, m.config, repl, tenantKey)
-	if err := input.validate(ctx, principal, validator, m.schemaManager); err != nil {
+	validator := validation.New(m.vectorRepo.Exists, m.config, repl)
+	if err := input.validate(ctx, principal, validator, m.schemaManager, tenantKey); err != nil {
```

Comment:
should we rename here `tenantKey` to `tenant` or `tenantName`?

Reply:
changed in renaming PR